core/src/main/java/io/confluent/rest/exceptions/WebApplicationExceptionMapper.java

```diff
@@ -32,6 +32,45 @@
   @Context
   HttpHeaders headers;
 
+  public static enum Status implements Response.StatusType {
```

Comment:
In order to account for other status codes besides 422, maybe just make this a class that can hold an arbitrary `statusCode`?

Reply:
I was not inclined to change the behavior  for other non-supported status codes, but as you think it can be changed for all, I have updated it.